process/block/interceptedBlocks/common_test.go

```diff
@@ -139,28 +108,6 @@ func TestCheckBlockHeaderArgument_NilShardCoordinatorShouldErr(t *testing.T) {
 	assert.Equal(t, process.ErrNilShardCoordinator, err)
 }
 
-func TestCheckBlockHeaderArgument_NilKeyGenShouldErr(t *testing.T) {
-	t.Parallel()
-
-	arg := createDefaultBlockHeaderArgument()
-	arg.KeyGen = nil
-
-	err := checkBlockHeaderArgument(arg)
-
-	assert.Equal(t, process.ErrNilKeyGen, err)
-}
-
-func TestCheckBlockHeaderArgument_NilSingleSignerShouldErr(t *testing.T) {
-	t.Parallel()
-
-	arg := createDefaultBlockHeaderArgument()
-	arg.SingleSigVerifier = nil
-
-	err := checkBlockHeaderArgument(arg)
-
-	assert.Equal(t, process.ErrNilSingleSigner, err)
-}
-
 func TestCheckBlockHeaderArgument_ShouldWork(t *testing.T) {
```

Comment:
add unit test for nil header sig verifier

Reply:
done.